main/acle.md

```diff
@@ -1072,6 +1074,33 @@ including the header:
 
 Including `<arm_sme.h>` also includes [`<arm_sve.h>`](#arm_sve.h).
 
+These macro features:
+`__ARM_ACLE`, `__ARM_FEATURE_FP16_SCALAR_ARITHMETIC`, `__ARM_FEATURE_BF16`,
+`__ARM_NEON`, `__ARM_FEATURE_SVE` and ` __ARM_FEATURE_SME`
+represent a compiler's ability to use the instructions associated
+with the feature via ACLE documented builtins and within inline assembly.
+They may also be influenced by source code mechanisms like `#pragma` or
+`attribute`, but users should not rely on this. They are defined for user
```

Comment:
this is badly phrased.  Do you mean here that users should not rely on #pragma or attribute or the feature.
macros?

If it is about the macros, I think we need to phrase it differently and explain that the presence of the macro does not guarantee that the architecture feature is always present at various stages of compilation or execution.

Reply:
Hi Maciej,
I re-wrote this paragraph and added a section at the end  of HEADER to explain headers + macros. 
Just in case the ACLE has a section: **Feature test macros**. It is  generic, it is about all predefined macros. 
The point here is that  the ACLE/I  want to explain that  headers protected by macro  does not imply we can always use intrinsics.